ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/remove_workload.yml

```diff
@@ -13,5 +13,5 @@
 # --------------------------------------------
 - name: remove_workload tasks complete
   debug:
-    msg: "Remove Workload tasks completed successfully."
+   msg: "Remove Workload tasks completed successfully."
```

Comment:
This doesn't look right... looks like it went from two space indent to one?

Reply:
Yeah.. its what the formatter wanted lol